src/web/jstl/compat/onedotzero/positiveFormatLocalizationContextI18NTestJava20Plus.jsp

```diff
@@ -0,0 +1,50 @@
+<%--
+
+    Copyright (c) 2003, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think the reference to `2003, 2021` is wrong since this is a new file even though it was derived from an existing file.  Perhaps just keep the `Copyright (c) 2024 Contributors to the Eclipse Foundation` which I think would agree with https://www.eclipse.org/projects/handbook/#ip-copyright-headers